DFC.App.FindACourse/Controllers/CourseController.cs

```diff
@@ -402,11 +402,11 @@ public async Task<IActionResult> SearchCourse(string searchTerm, string townOrPo
         {
             logService.LogInformation($"{nameof(this.SearchCourse)} has been called");
 
-            if (string.IsNullOrWhiteSpace(searchTerm) && string.IsNullOrWhiteSpace(townOrPostcode))
-            {
-                logService.LogInformation($"{nameof(searchTerm)} and {nameof(townOrPostcode)} is null or whitespace");
-                return Body();
-            }
+            //if (string.IsNullOrWhiteSpace(searchTerm) && string.IsNullOrWhiteSpace(townOrPostcode))
```

Comment:
Remove commented out code

Reply:
This has been resolved in 57.2 which will be merged in next week

Reply:
Thanks Amandeep. Yes. its removed on other feature branch - AD-157796 which will be released on 57.2